template.tpl

```diff
@@ -54,6 +54,14 @@ ___TEMPLATE_PARAMETERS___
     "defaultValue": true,
     "help": "When this checkbox is enabled the tracker will send a pageview event anytime the tag is included.\nDisabling this field might be useful e.g. when Leadfeeder tracker is used with Single Page Applications.\nYou can read more about SPA tracking \u003ca href\u003d\"https://help.leadfeeder.com/en/articles/4519005-customize-your-site-tracking-with-leadfeeder-javascript-api\" target\u003d\"_blank\"\u003eLeadfeeder Help Center\u003c/a\u003e",
     "alwaysInSummary": true
+  },
+  {
+    "type": "CHECKBOX",
+    "name": "enable_debug",
+    "checkboxText": "Enable debug script",
+    "simpleValueType": true,
+    "help": "When this checkbox is enabled the tracker adds debug logs.\nDisabling this field reduces the size of the Leadfeeder script.\n",
```

Comment:
```suggestion
    "help": "When this checkbox is enabled the tracker will write verbose logs to the browser's developer console.\nWe suggest turning this on only while testing your tracking script in the GTM preview mode. Disabling it reduces the size of the Leadfeeder tracking script and is a suggested option for the production websites.\n",
```

Reply:
Good suggestion @okulik. We could even consult the copy with Vito, but this looks good to me.